src/AccessibilityInsights/MainWindowHelpers/ControlHelper.cs

```diff
@@ -261,11 +263,20 @@ private void UpdateTitleString()
 
         /// <summary>
         /// Set version text
-        /// Release Type - Version - UIAccess state
+        /// [(ReleaseChannel) - ]Version - UIAccess state
         /// </summary>
         private void UpdateVersionString()
         {
-            StringBuilder sb = new StringBuilder(string.Format(CultureInfo.InvariantCulture, Properties.Resources.UpdateVersionStringVer, AccessibilityInsights.Core.Misc.Utility.GetAppVersion()));
+            StringBuilder sb = new StringBuilder();
+
+            ReleaseChannel? channel = ConfigurationManager.GetDefaultInstance()?.AppConfig?.ReleaseChannel;
```

Comment:
This means that we will display the currently configured channel, which may not be the actual installed/running channel in rare cases. Is this ok for our users? @risanaka @neocln

Reply:
Are you thinking of cases where someone should be configured for Insider or Canary, but some issue prevents the config from being read? That would look like Production, even though it's ahead of the current build for Production. That feels like the same issue as someone installing a Canary build directly from the server, but not actually setting the Canary channel. If we can live with the latter case, then it seems like we would also be able to live with the former case.